apps/iframe/src/connectService.ts

```diff
@@ -58,7 +58,7 @@ export class ConnectService extends IframeService<DIDProviderMethods> {
     this.cancel = cancel
     this.errorCb = errorCb
     this.userRequestHandler = userRequestHandler
-    this.ceramic = new CeramicClient(CERAMIC_API)
+    this.ceramic = new CeramicClient(CERAMIC_API, { syncInterval: 30000 })
```

Comment:
I'm not familiar with `syncInterval`, what unit of value is it using please?
If it's milliseconds as in JS timers, it seems like this change would sync every 30 seconds, not 30 minutes as stated in the PR title?

Reply:
My bad. Good catch.